classes/UpgradeSelfCheck.php

```diff
@@ -465,4 +480,17 @@ public function runPrestaShopCoreChecks()
 
         return true;
     }
+
+    /**
+     * Check if the current php version is compatible whith PrestaShop verion to Upgrade
+     *
+     * @return array
+     */
+    public function phpCompatibleVersions()
+    {
+        $phpCompatibleVersion = self::PHP_PS_VERSIONS[substr($this->upgrader->version_num, 0, 5)];
+
+        return $phpCompatibleVersion;
```

Comment:
```suggestion
        return self::PHP_PS_VERSIONS[substr($this->upgrader->version_num, 0, 5)];
```

Using substr is not the right way to find the version, use version_compare instead, because it will not work with 8.0 or 1.6 :thinking: